config/routes.rb

```diff
@@ -6,6 +6,7 @@
       post "accept" => "short_url_requests#accept"
       get "new_rejection" => "short_url_requests#new_rejection"
       post "reject" => "short_url_requests#reject"
+      delete "remove" => "short_url_requests#remove"
```

Comment:
I originally made this a GET route 'cause I was under the mistaken impression that DELETE didn't allow any other manipulations apart from, say,  `@short_url_request.destroy`. It all still works with the standard `destroy` action, so I've amended it. Thanks for prompting that!